.github/workflows/build_and_test_workflow.yml

```diff
@@ -25,18 +25,18 @@ env:
   OSD_SNAPSHOT_SKIP_VERIFY_CHECKSUM: true
 
 jobs:
-  build-lint-test:
+  build-lint-test-linux:
     runs-on: ubuntu-latest
     container:
       image: docker://opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
```

Comment:
Should this image be also bumped to integtest-v3 based on the below version bump to v3 or is the same image?

Reply:
Nah. that is a custom image of ours. Those `v3`s are stuff that run on GH. However, we should figure out what blockers we have to using standard GH images.